### PR TITLE
Snippets: fix misnamed snippets in `archetypes/`

### PR DESCRIPTION
**Pull request by teh-cmc**
Archetype snippets should follow the model `{archetype_name_snake_case}_{snippet_name}`.

[Index rendered](https://github.com/rerun-io/rerun/blob/cmc/snippets_index_4_cleanup_data/docs/snippets/INDEX.md)

Reasons:
A) Consistency.
B) Snippets that contain the name of an archetype bubble up in the snippet index (when grepping for Points3D, you are likely more interested in points3d_simple vs. transform_hierarchy_mega_test_of_deth, even though both of them reference Points3D).

* DNM: requires #8386

### CODE REVIEW

**Comment by teh-cmc** [comment]
Link checking failures will pass once this is on `main`.

**Comment by teh-cmc** [comment]
@rerun-bot full-check

File: crates/store/re_types/definitions/rerun/archetypes/line_strips2d.fbs [hunk 1]
```diff
@@ -4,10 +4,10 @@ namespace rerun.archetypes;
 
 /// 2D line strips with positions and optional colors, radii, labels, etc.
 ///
-/// \example archetypes/line_strip2d_simple !api image="https://static.rerun.io/line_strip2d_simple/c4e6ce937544e66b497450fd64ac3ac2f244f0e1/1200w.png"
+/// \example archetypes/line_strips2d_simple !api image="https://static.rerun.io/line_strip2d_simple/c4e6ce937544e66b497450fd64ac3ac2f244f0e1/1200w.png"
 /// \example archetypes/line_segments2d_simple !api image="https://static.rerun.io/line_segment2d_simple/53df596662dd9ffaaea5d09d091ef95220346c83/1200w.png"
```
Comment (Wumpf) [inline crates/store/re_types/definitions/rerun/archetypes/line_strips2d.fbs:8]: hmmm the arcehtype is still `line_strip2d`, so maybe this one whould be `line_strip2d_segments_simple`?

File: crates/store/re_types/definitions/rerun/archetypes/line_strips2d.fbs [hunk 2]
```diff
@@ -4,10 +4,10 @@ namespace rerun.archetypes;
...
-/// \example archetypes/line_strip2d_batch image="https://static.rerun.io/line_strip2d_batch/c6f4062bcf510462d298a5dfe9fdbe87c754acee/1200w.png"
-/// \example archetypes/line_strip3d_ui_radius title="Lines with scene & UI radius each" image="https://static.rerun.io/line_strip2d_ui_radius/d3d7d5bd36278564ee37e2ed6932963ec16f5131/1200w.png
+/// \example archetypes/line_strips2d_batch image="https://static.rerun.io/line_strip2d_batch/c6f4062bcf510462d298a5dfe9fdbe87c754acee/1200w.png"
+/// \example archetypes/line_strips3d_ui_radius title="Lines with scene & UI radius each" image="https://static.rerun.io/line_strip2d_ui_radius/d3d7d5bd36278564ee37e2ed6932963ec16f5131/1200w.png
```
Comment (Wumpf) [inline crates/store/re_types/definitions/rerun/archetypes/line_strips2d.fbs:10]: not your fault in the first place, but this is a 3d snippet that shouldn't be here!

File: crates/store/re_types/definitions/rerun/archetypes/line_strips3d.fbs [hunk 3]
```diff
@@ -4,10 +4,10 @@ namespace rerun.archetypes;
 
 /// 3D line strips with positions and optional colors, radii, labels, etc.
 ///
-/// \example archetypes/line_strip3d_simple !api title="Simple example" image="https://static.rerun.io/line_strip3d_simple/13036c0e71f78d3cec37d5724f97b47c4cf3c429/1200w.png"
+/// \example archetypes/line_strips3d_simple !api title="Simple example" image="https://static.rerun.io/line_strip3d_simple/13036c0e71f78d3cec37d5724f97b47c4cf3c429/1200w.png"
 /// \example archetypes/line_segments3d_simple !api title="Many individual segments" image="https://static.rerun.io/line_segment3d_simple/aa800b2a6e6a7b8e32e762b42861bae36f5014bb/1200w.png"
```
Comment (Wumpf) [inline crates/store/re_types/definitions/rerun/archetypes/line_strips3d.fbs:8]: as above

**Review by Wumpf — APPROVED** [review]
found some small issues, otherwise lgtm!

**Pull request merged**